aws_kms_policies/aws_cmk_key_rotation.yml

```diff
@@ -113,3 +113,26 @@ Tests:
         "ValidTo": null,
         "Policy": "{JSON policy document see https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html for details}"
       }
+  -
+    Name: Key Not Eligible for Roration
+    ExpectedResult: true
+    Resource:
+      {
+        "KeyRotationEnabled": true,
```

Comment:
make this false or null just to ensure it is still ignored?

Reply:
changed to null, still looks good 🎉